scripts/arangodb-docker-container-setup/docker-container-scripts/js/setup.js

```diff
@@ -52,4 +53,26 @@ collectionNames.forEach(collectionName => {
   else {
     print(`Collection "${collectionName}" could not be created`);
   }
-});
+}
+
+print('Edge Collections')
+const edgeCollectionName = 'resource_connections';
```

Comment:
We should use the key that is used in the data for this. So if you want to name it `resource_connections`, please update `validateTestData.spec.ts` to write the connections to a key `resource_connections` (~~connections~~) in the JSON. Otherwise, change the collection name to `connections`. Your choice.

Reply:
Ok, will do.  I like resource_connections just because edges seems too generic.